plugins/inputs/kube_inventory/node.go

```diff
@@ -0,0 +1,56 @@
+package kube_inventory
+
+import (
+	"context"
+
+	"github.com/ericchiang/k8s/apis/core/v1"
+
+	"github.com/influxdata/telegraf"
+)
+
+func collectNodes(ctx context.Context, acc telegraf.Accumulator, ki *KubernetesInventory) {
+	list, err := ki.client.getNodes(ctx)
+	if err != nil {
+		acc.AddError(err)
+		return
+	}
+	for _, n := range list.Items {
+		if err = ki.gatherNode(*n, acc); err != nil {
+			acc.AddError(err)
+			return
+		}
+	}
+}
+
+func (ki *KubernetesInventory) gatherNode(n v1.Node, acc telegraf.Accumulator) error {
+	fields := map[string]interface{}{}
+	tags := map[string]string{
+		"node_name": *n.Metadata.Name,
+	}
+
+	for resourceName, val := range n.Status.Capacity {
+		switch resourceName {
+		case "cpu":
+			fields["capacity_cpu_cores"] = atoi(val.GetString_())
+		case "memory":
+			fields["capacity_memory_bytes"] = val.GetString_()
```

Comment:
We need to convert these strings to an integer.

Reply:
We'll need a parser for that then, as memory_bytes has values like `125715504Ki`